sdk/src/main/java/io/opentdf/platform/sdk/TDF.java

```diff
@@ -495,22 +504,46 @@ private void fillInPublicKeyInfo(List<Config.KASInfo> kasInfoList, SDK.KAS kas)
     }
 
     public Reader loadTDF(SeekableByteChannel tdf, Config.AssertionConfig assertionConfig, SDK.KAS kas) throws NotValidateRootSignature, SegmentSizeMismatch,
-            IOException, FailedToCreateGMAC, JOSEException, ParseException, NoSuchAlgorithmException, DecoderException {
+            IOException, FailedToCreateGMAC, JOSEException, ParseException, NoSuchAlgorithmException, DecoderException, SplitKeyException {
```

Comment:
I'd recommend against adding a new checked `Exception` here since it will count as a semver breaking change. Instead extend an existing checked exception or let `SplitKeyException` extend (hopefully some more specific subclass of) `RuntimeException`